autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/ScheduleListItem/ScheduleListItem.tsx

```diff
@@ -59,13 +59,23 @@ const ScheduleListItem: React.FC<ScheduleListItemProps> = ({ index, onDetailsCli
     </ListItemSecondaryAction>
   );
 
-  const scheduleSections = sectionsForSchedule(schedule).map((sec: Section) => (
-    <span key={sec.id} className={styles.sectionLabelRow}>
-      <ColorBox color={meetingColors.get(sec.subject + sec.courseNum)} />
-      {`${sec.subject} ${sec.courseNum}-${sec.sectionNum}`}
-      <br />
-    </span>
-  ));
+  const scheduleSections = sectionsForSchedule(schedule).map((sec: Section) => {
+    const color = !sec.asynchronous && meetingColors.get(sec.subject + sec.courseNum);
```

Comment:
This is like an alternative form of a ternary where the false case is undefined/null right?

Reply:
Yeah although I probably should've just explicitly set it to undefined to make it more clear

Reply:
Yeah I'd say so, honestly I don't really like this syntax that much in this scenario (I'm more inclined to favor it when it's in actual JSX)